CoreFoundation/Base.subproj/CFPriv.h

```diff
@@ -575,6 +575,12 @@ CF_INLINE CFAbsoluteTime _CFAbsoluteTimeFromFileTimeSpec(struct timespec ts) {
     return (CFAbsoluteTime)((CFTimeInterval)ts.tv_sec - kCFAbsoluteTimeIntervalSince1970) + (1.0e-9 * (CFTimeInterval)ts.tv_nsec);
 }
 
+// For whatever reason, including <time.h> is not defining time_t on FreeBSD
```

Comment:
I see it defined in https://github.com/freebsd/freebsd/blob/master/include/time.h — can you investigate it?